meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,134 @@
+# v0.29.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using this version in production. This is only for test purposes. Please, feel free to [report bugs](https://github.com/meilisearch/meilisearch/issues/new/choose) and [give feedback about the new feature introductions](https://github.com/meilisearch/product/discussions).
+
+In this release, we integrated a demanded feature: requiring all query terms to be present in search results. Some users have been stuck in the pre-0.21 versions because of this.
+We also improved the indexing speed
```

Comment:
Minor wording change for better text flow, plus integrating that hanging sentence into the paragraph.

```suggestion
This release includes a much requested feature: an option to require all query terms to be present in search results. The absence of this feature prevented some users from upgrading to newer Meilisearch versions. Additionally, v0.29 contains several indexing speed improvements.
```